frontend/src/pages/associations/elections/ActiveList.tsx

```diff
@@ -0,0 +1,246 @@
+    const { data, status, error } = useBetterQuery<Election[]>(
+        "elections.list",
+        api.elections.list,
+        associationId,
+        activeStatus.Current
+    );
+
+    if (status === "loading") return <LoadingAssociation />;
+    else if (status === "error") return `Something went wrong: ${error}`;
+    else if (data) {
+        return (
+            <>
+                <PageTitle className={"mt-6"}>Élections en cours</PageTitle>
+                <Row>
+                    {data.map(election => {
+                        return (
+                            <Col md={5} key={election.id}>
+                                <ElectionCard election={election} association={association}/>
+                            </Col>
+                        );
+                    })}
+                </Row>
+            </>
+        )
+    }
+};
+
+const ElectionCard = ({ election, association }) => {
+    let editButton;
+    if (association.myRole.electionPermission) {
+        editButton = (
+            <Link
+                to={`/associations/${association.id}/elections/${election.id}/edit`}
+                className={"float-right mt-0"}
+            >
+                <i className={"fe fe-edit"}/> Modifier
+            </Link>
+        );
+    } else {
+        editButton = (<></>)
+    }
+    const startsAt = new Date(election.startsAt);
+    const endsAt = new Date(election.endsAt);
+    return (
+        <Card>
+            <Card.Body>
+                <Row>
+                    <Col xs={0}>
+                        <h4>{election.name}</h4>
+                    </Col>
+                    <Col>
+                        {editButton}
+                    </Col>
+                </Row>
+                <Row>
+                    <p>Cette élection, ouverte le {startsAt.toLocaleDateString()} à {startsAt.toLocaleTimeString()}, fermera le {endsAt.toLocaleDateString()} à {endsAt.toLocaleTimeString()}. Les résultats seront immédiatement publiés.</p>
+                    <p>Vous avez le droit à un maximum de {election.maxChoicesPerBallot} choix</p>
+                </Row>
+                <Row>
+                    <Vote election={election} association={association}/>
+                </Row>
+                <Row>
+                    <ListOfVotersButtonModal election={election}/>
+                </Row>
+            </Card.Body>
+        </Card>
+    )
+};
+
+const Vote = ({ election, association, ...props }) => {
+    const isRegisterd = election.isRegistered;
+    const hasVoted = election.hasVoted;
+    const disabled = hasVoted || !isRegisterd;
+    const maxChoicesPerBallot = parseInt(election.maxChoicesPerBallot);
+
+    const [selected, setSelected] = useState<number[]>([]);
+    const [showVoteConfirmation, setShowVoteConfirmation] = useState<boolean>(false);
+    const newToast = useContext(ToastContext);
+
+    const disabledVoteButton = disabled || selected.length===0;
+
+
+    const handleClick = (choice: Choice) => {
```

Comment:
Please add one or two comments to quickly explain what is going on here

Reply:
done